packages/verification-vue-component/src/locales/de.json

```diff
@@ -65,12 +65,16 @@
                 "signedVia": "Unterzeichnet über",
                 "issuerAddress": "Public Key des Ausstellers",
                 "documentHash": "Dokumenten-Hash",
+                "txHash": "Transaction hash",
                 "registrationDate": "Ausstelldatum & -Zeit",
                 "registrationTransaction": "Ausstellungstransaktion",
                 "revocationTransaction": "Annullierungstransaktion",
                 "revocationDate": "Annullierungsdatum & -Zeit",
                 "smartContractAddress": "Smart Contract-Adresse",
-                "signatureTransactions": "Unterzeichnungstranskationen",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download signature",
```

Comment:
@stephalba could you please translate the changed labels to german?

Reply:
Done

Reply:
Thanks